modules/exploits/windows/misc/ais_esel_server_rce.rb

```diff
@@ -0,0 +1,149 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::MSSQL_COMMANDS
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'AIS logistics ESEL-Server unauthenticated Payload
+        Execution via SQL Injection',
```

Comment:
Fit this on one line please.

Consider: `AIS logistics ESEL-Server Unauth SQL Injection RCE`